Packs/SlackITAdmin/Integrations/Slack_IT_Admin/Slack_IT_Admin.py

```diff
@@ -0,0 +1,724 @@
+                                      headers=["brand", "instanceName", "success", "active", "id", "username", "email",
+                                               "errorCode", "errorMessage", "details"],
+                                      removeNull=True)
+    return (
+        readable_output,
+        outputs,
+        generic_iam_context.data
+    )
+
+
+def get_group(client, args):
+    group_id = args.get('groupId')
+    group_name = args.get('groupName')
+
+    if not (group_id or group_name):
+        return_error("You must supply either 'groupId' or 'groupName")
+    if not group_id:
+        group_id = client.get_group_id(group_name)
+
+    res = client.get_group_by_id(group_id)
+    res_json = res.json()
+
+    if res.status_code != 200:
+        return_error(f"Error in API call. Status Code: [{res.status_code}]. Error Response: {res_json}")
+
+    members = res_json.get('members')
+    outputs = {
+        'Slack.Group(val.Id && val.Id === obj.Id)': {
+            'Id': group_id,
+            'DisplayName': res_json.get('displayName'),
+            'Members': members
+        }
+    }
+
+    readable_output = tableToMarkdown(name=f"Slack Group Members: {res_json.get('displayName')} [{group_id}]",
+                                      t=members)
+
+    return (
+        readable_output,
+        outputs,
+        res_json
+    )
+
+
+def delete_group_command(client, args):
+    group_id = args.get('groupId')
+    group_name = args.get('groupName')
+
+    if not (group_id or group_name):
+        return_error("You must supply either 'groupId' or 'groupName")
+    if not group_id:
+        group_id = client.get_group_id(group_name)
+
+    res = client.delete_group(group_id)
+
+    if res.status_code != 204:
+        error_json = res.json()
+        return_error(f"Error in API call. Status Code: [{res.status_code}]. Error Response: {error_json}")
+
+    readable_output = f'Slack Group: "{group_id}" was deleted successfully'
+    return (
+        readable_output,
+        {},
+        None)
+
+
+def create_group_command(client, args):
+
+    group_data = {'schemas': [SLACK_SCIM_CORE_SCHEMA_KEY], 'displayName': args.get('groupName')}
+    members = []
+    member_ids = args.get('memberIds')
+    if member_ids:
+        if type(member_ids) is not list:
+            try:
+                member_ids = json.loads(member_ids)
+            except JSONDecodeError:
+                return_error("memberIds is not a valid list")
+        for member_id in member_ids:
+            members.append({"value": member_id})
+
+    group_data['members'] = members
+    res = client.create_group(group_data)
+    res_json = res.json()
+
+    if res.status_code not in [200, 201]:
+        return_error(f"Error in API call. Status Code: [{res.status_code}]. Error Response: {res_json}")
+
+    readable_output = f'Slack Group "{res_json.get("id")}" created successfully'
+    return (
+        readable_output,
+        {},
```

Comment:
please add the data to the context as well (and update in the yml the context output)